server/__tests__/server.test.js

```diff
@@ -81,12 +81,13 @@ describe("shopify-app-node server", async () => {
   });
 
   test("renders toplevel auth page", async () => {
+    const host = btoa("test-shop.myshopify.com/admin");
     const response = await request(app)
```

Comment:
SUPER NIT: We prefer [AAA spacing in tests](https://shopify.dev/internal/development/tips#writing-tests) 

[public](https://github.com/testdouble/contributing-tests/wiki/Arrange-Act-Assert) [links](https://thoughtbot.com/blog/four-phase-test) [for the curious](https://ruby-style-guide.shopify.dev/#testing)

```suggestion
    const host = btoa("test-shop.myshopify.com/admin");

    const response = await request(app)
```